@kiva/kv-components/vue/KvClassicLoanCard.vue

```diff
@@ -453,12 +457,12 @@ export default {
 
 			if (!this.categoryPageName) {
 				if (categories.ecoFriendly
-					&& !callouts.filter((c) => c.toUpperCase() === categories.ecoFriendly.toUpperCase()).length) {
+					&& !callouts.find((c) => c === ECO_FRIENDLY_KEY || c === SUSTAINABLE_AG_KEY)) {
```

Comment:
Do we still need case insensitive checking here?

Reply:
Looks like test is passing, but good for consistency. ty